src/modeler/AutoRest.Swagger/Validation/BodyPropertiesNamesCamelCase.cs

```diff
@@ -41,7 +41,7 @@ public class BodyPropertiesNamesCamelCase : TypedRule<Dictionary<string, Operati
         /// <summary>
         /// The severity of this message (ie, debug/info/warning/error/fatal, etc)
         /// </summary>
-        public override Category Severity => Category.Error;
+        public override Category Severity => Category.Warning;
```

Comment:
Oh, I see. I agree that this is a naming convention rule, I'm not sure we want to turn this one into a Warning yet though. I think the naming ones we want to turn into Warnings are the ones about operation names, I'd start with those and we can evaluate this one if needed afterwards.